docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -161,7 +161,7 @@ Depending on your role on the Git provider you will have different permissions o
     </tr>
     <tr>
       <td rowspan="2">Bitbucket Cloud and Bitbucket Server</td>
-      <td>Read, Write<sup><a href="#note-4">4</a></td>
+      <td>Read, Write<sup><a href="#note-3">4</a></td>
```

Comment:
Change the number to 3

Reply:
Love this attention to detail! :female_detective: :bow: